src/main/resources/assets/js/app/highchart.es6

```diff
@@ -119,20 +119,18 @@ export function init() {
           },
         }
 
-        // Only show plotOption marker on last data element
-        if (canvas.data('type') === 'line') {
-          config.series.forEach(function (series) {
-            const lastIndex = series.data.length - 1
-            series.data.forEach(function (data, index) {
-              series.data[index] = {
-                y: parseFloat(data),
-                marker: {
-                  enabled: index === lastIndex,
-                },
-              }
-            })
-          })
-        }
+        // Only show plotOption marker on last data element / single data point series
```

Comment:
since this only applies to line charts the code should be wrapped in if (canvas.data('type') === 'line') { like before